tests/e2e/s3/Vagrantfile

```diff
@@ -44,6 +44,8 @@ def provision(vm, role, role_num, node_num)
         node-external-ip: #{NETWORK_PREFIX}.100
         flannel-iface: eth1
         cluster-init: true
+        etcd-snapshot-schedule-cron: '*/1 * * * *'
```

Comment:
This might make the test flakey, as the number of snapshots present will depend on whether or not the test happens to run over 1 or more crown intervals. I don't have a good suggestion on how to control for that though.